backend/users/serializers/token.py

```diff
@@ -7,7 +7,7 @@
 
 
 class BearerTokenSerializer(serializers.ModelSerializer):
-    expires_at = serializers.DateTimeField(default_timezone=timezone.utc, allow_null=True)
+    expires_at = serializers.DateTimeField(default_timezone=timezone.utc, allow_null=False)
```

Comment:
If a field as a default, you have to set `required=True` to avoid receiving null values

Reply:
`required=True` with the `default=timezone.now` at the model level does not prevent from calling with no value